api/main.py

```diff
@@ -407,7 +407,8 @@ def get_auction_tweets(self, user_id):
                     'created_at': tweet['created_at'],
                     'auction_key': auction_key,
                     'photos': [m for m in response_json['includes']['media']
-                        if m['media_key'] in media_keys and m['type'] == 'photo'],
+                        # media animated_gifs, photos, video - ignore video
+                        if m['media_key'] in media_keys and m['type'] != 'video'],
```

Comment:
I think we agreed to include images and gifs rather than exclude video. What if they add a new media type suddenly? It has to be whitelist-based, not blacklist-based!